<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWidget</class>
 <widget class="QWidget" name="ResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>651</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainQHBoxLayout">
    <item>
     <widget class="QScrollArea" name="imagesQScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="imagesQWidgetContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>554</width>
         <height>507</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>561</width>
          <height>511</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="imagesQVBoxLayout"/>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="detailsQPushButton">
        <property name="text">
         <string>Details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="returnQPushButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
